core/src/main/java/org/jdbi/v3/BuiltInArgumentFactory.java

```diff
@@ -92,6 +99,15 @@
         register(map, Timestamp.class, Types.TIMESTAMP, PreparedStatement::setTimestamp);
         register(map, URL.class, Types.DATALINK, PreparedStatement::setURL);
         register(map, URI.class, Types.VARCHAR, stringifyValue(PreparedStatement::setString));
+
+        Calendar c = Calendar.getInstance();
+        Supplier<Calendar> cal = () -> (Calendar) c.clone();
+        register(map, Instant.class, Types.TIMESTAMP, (p, i, v) -> p.setTimestamp(i, Timestamp.from(v), cal.get()));
```

Comment:
If the tests pass without it, that works for me. I was just following `dropwizard-jdbi`'s lead, not fully understanding why they used the calendar in certain cases.